deployment/network-operator/README.md

```diff
@@ -114,9 +114,13 @@ to be associated with the resource.
 ```
 resources:
     - name: rdma_shared_device_a
-      devices: [enp5s0f0]
+      vendors: [15b3]
```

Comment:
the description of this section should be updated

Reply:
Updated